Flow.Launcher.Core/Plugin/JsonRPCPlugin.cs

```diff
@@ -69,7 +69,7 @@ public List<Result> LoadContextMenus(Result selectedResult)
         private static readonly JsonSerializerOptions options = new()
         {
             PropertyNameCaseInsensitive = true,
-            IgnoreNullValues = true,
+            DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull,
```

Comment:
Basically because of this?

Reply:
exactly, but the change (`IgnoreNullValues` is deprecated) should have been the same. Looking into the alternative, but might just give up and stay with the deprecated property.

Reply:
What's the difference between this two property? If they are supposed to replace each other, there shouldn't be an issue with that.

Reply:
Let's remove this property. I see no reason we want to ignore null/default property.

Reply:
If I remember correctly, the original usage is to ignore null value when deserialize. Do we have a similar api for that?

Reply:
I take a look on the test case, and it is testing whether different naming case will create same result, which should be responsible for `PropertyNameCaseInsensitive` but not `IgnoreNullValue`. I am quite curious what's causing that.

Reply:
My test result is surprising. It will not be `null` only when we have the `IgnoreNullValue` property set to true. I have limited clues about it.

Reply:
The new property only kicks in during serialization, which means setting an explicit null, whereas the previous value ignores the null and seems to auto-set the default value (which in this case is an empty List for [`Result`](https://github.com/Flow-Launcher/Flow.Launcher/blob/dev/Flow.Launcher.Core/Plugin/JsonPRCModel.cs#L44)).

This is fixable, though not sure it gets us anything.

Reply:
The weird thing is only the second result is null. I don't understand what's causing that.

Reply:
I try to take a debug but there seems like it cannot deserialize the second result at all.

Reply:
I think it's because the second result matches case exactly. It just seems... broken, which is why I propose not fixing this for now.

Reply:
That seems reasonable. Though I am still stuck at the reason of the null here....Maybe it's somewhat because of how I handle the stream or what? Let's investigate that in the future.